src/main/scala/cromwell/engine/backend/local/LocalEngineFunctions.scala

```diff
@@ -15,7 +15,9 @@ class LocalEngineFunctions(executionContext: TaskExecutionContext) extends Engin
   override protected def read_int(params: Seq[Try[WdlValue]]): Try[WdlInteger] = {
```

Comment:
This reminds me that while I was going over some stuff w/ kristian i'm pretty sure i saw a couple of cases where things expecting Trys are sitting on top of code which can throw exceptions. sigh.